setup_plausible.sh

```diff
@@ -15,7 +15,7 @@ docker compose exec -T "$PLAUSIBLE_DB_SERVICE_NAME" /bin/bash -c "psql -U deploy
 	INSERT INTO api_keys
 	  (id, user_id, name, key_prefix, key_hash, inserted_at, updated_at, scopes, hourly_request_limit)
 	VALUES
-	  (1, 1, 'Development', 'aaaaaa', '332015ffc9f0e1f475c0fadb1e1a14d2eb09774249f71961f5a2c477efc0a0fc', now(), now(), '{sites:provision:*}', 1000)
+	  (1, 1, 'Development', 'aaaaaa', '68e5212e2a10fb0b622250e2189f86051da4b908c6e82f343ee3260c05cf7082', now(), now(), '{sites:provision:*}', 1000)
```

Comment:
It would be pretty nice if we could generate this in this script, instead of needing to maintain it by hand.

If you set `api_key` and then `source docker/plausible/env.docker`, I think you could replace the key literal here with a call to [Postgres's crypto functions](https://www.postgresql.org/docs/current/pgcrypto.html):

```suggestion
	  (1, 1, 'Development', 'aaaaaa', encode(digest('$SECRET_KEY_BASE$api_key', 'sha1'), 'hex'), now(), now(), '{sites:provision:*}', 1000)
```

Not strictly necessary for this PR, but it would make changes to `SECRET_KEY_BASE` a lot more "arbitrary" and require less work both when testing a specific value or changing the default value again (if ever needed in the future).

Reply:
Hi @sarayourfriend This makes sense, let me update.

Reply:
This has now been updated, thank you for the  suggestion @sarayourfriend .